### PR TITLE
Fat 44381/input

### PR DESCRIPTION
**Pull request by MaximZybo**

### CODE REVIEW

File: src/components/InputPassword/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      secureTextEntry={areSymbolsHidden}
+      placeholder="Enter your password"
+      errorText={errorText}
+      autoCapitalize="none">
+      <Pressable
+        disabled={isDisabled}
+        style={styles.buttonHide}
+        hitSlop={30}
+        onPress={toggleSymbolsVisibility}>
+        {areSymbolsHidden && <HideEye width={14} color={iconColor} />}
```
Comment (MaximZybo) [inline src/components/InputPassword/index.tsx:44]: {areSymbolsHidden ? (
          HideEye
        ) : (
          Clocks
        )}

File: src/components/InputPassword/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        {!areSymbolsHidden && <Clocks width={14} color={iconColor} />}
```
Comment (MaximZybo) [inline src/components/InputPassword/index.tsx:45]: If there is an icon, right padding should be bigger
image

File: src/screens/Home/index.tsx [hunk 3]
```diff
@@ -1,105 +1,98 @@
...
     flex: 1,
-    backgroundColor: COLORS.baseLight80,
-  },
-  buttonsContainer: {
-    flexDirection: 'row',
-    justifyContent: 'space-between',
-  },
-  buttonLeft: {
-    flex: 1,
+    backgroundColor: 'white',
```
[the remark this answers is not in the document]
Comment (mariia-kiliushina) [reply src/screens/Home/index.tsx:85]: Ok, done

File: src/components/Input/index.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  containerStyle,
+  ...props
+}: Props) => {
+  const [text, setText] = useState('');
+  const [isFocused, setIsFocused] = useState(false);
+
+  const isDisabled = type === 'disabled';
+
+  const labelTextStyle = [
+    {color: isDisabled ? COLORS.neutral300 : COLORS.neutral900},
```
[the remark this answers is not in the document]
Comment (mariia-kiliushina) [reply src/components/Input/index.tsx:40]: Done

**Pull request merged**